lib/Listeners/FilesLoadAdditionalScriptsListener.php

```diff
@@ -33,6 +33,9 @@
 use OCP\IInitialStateService;
 use OCP\IUserSession;
 
+/**
+ * @implements IEventListener<Event>
```

Comment:
Using the generic Event class here since the OCA events are not available for static analysis